DigitalLearningSolutions.Web/Views/RegisterAdmin/Summary.cshtml

```diff
@@ -54,17 +54,30 @@
       </div>
       <div class="nhsuk-summary-list__row">
         <dt class="nhsuk-summary-list__key">
-          Email
+          Primary email
         </dt>
         <dd class="nhsuk-summary-list__value">
-          @Model.Email
+          @Model.PrimaryEmail
         </dd>
         <dd class="nhsuk-summary-list__actions">
           <a asp-action="PersonalInformation">
             Change<span class="nhsuk-u-visually-hidden"> email address</span>

```

Comment:
This needs the bug fix from https://github.com/TechnologyEnhancedLearning/DLSV2/pull/1163/files#diff-7364ba87f04e1d5228f6021f00d83e51e28e9d646abf0483f8b5353a036b9ab0

Reply:
done